package/osx/CMakeLists.txt

```diff
@@ -3,19 +3,19 @@
 if (RSTUDIO_PACKAGE_BUILD)
    INSTALL(CODE "
      list (APPEND CODESIGN_TARGETS \"\${CMAKE_INSTALL_PREFIX}/RStudio.app\")
-  
+
      file(GLOB_RECURSE CODESIGN_PLUGINS \"\${CMAKE_INSTALL_PREFIX}/RStudio.app/Contents/plugins\")
      list (APPEND CODESIGN_TARGETS \${CODESIGN_PLUGINS})
 
      file(GLOB_RECURSE CODESIGN_FRAMEWORKS \"\${CMAKE_INSTALL_PREFIX}/RStudio.app/Contents/Frameworks\")
      list (APPEND CODESIGN_TARGETS \${CODESIGN_FRAMEWORKS})
-  
+
      file(GLOB_RECURSE CODESIGN_MACOS \"\${CMAKE_INSTALL_PREFIX}/RStudio.app/Contents/MacOS\")
      list (APPEND CODESIGN_TARGETS \${CODESIGN_MACOS})
-  
+
      foreach(CODESIGN_TARGET \${CODESIGN_TARGETS})
-        execute_process(COMMAND codesign \"-s\" \"Developer ID Application: RStudio Inc. (FYF2F5GFX4)\" \"--deep\" \"-i\" \"org.rstudio.RStudio\" \"\${CODESIGN_TARGET}\")
-     endforeach()   
+        execute_process(COMMAND codesign \"-s\" \"8A388E005EF927A09B952C6E71B0E8F2F467AB26\" \"--deep\" \"-i\" \"org.rstudio.RStudio\" \"\${CODESIGN_TARGET}\")
```

Comment:
As far as I can tell from the documentation, this is actually the more compatible way to codesign with a specific key -- so it should be safe to use in both build systems).

A good test would be: `codesign -s 8A388E005EF927A09B952C6E71B0E8F2F467AB26 --deep -v -i org.rstudio.RStudio $(mktemp)` on the existing build node. It should succeed if the existing key can be looked up with this identifier, and fail if it can't. If it can't we can discuss an alt way of doing this!